configureWorkspace/configure.ts

```diff
@@ -20,29 +20,25 @@ CMD ${cmd}`;
         case 'go':
 
             return `
-# golang:onbuild automatically copies the package source, 
-# fetches the application dependencies, builds the program, 
-# and configures it to run on startup 
-FROM golang:onbuild
-LABEL Name=${serviceName} Version=${version} 
+# golang:alpine is based on Alpine Linux and as a result contains a very
+# slim base image. This dockerfile configures ssh, bash for remote access
+# and git for running go. Customize the Dockerfile as required.
+FROM golang:alpine
+WORKDIR /go/src/app
+COPY . .
+RUN apk update && apk upgrade && apk add --no-cache bash git openssh
```

Comment:
not all would agree with including SSH, see e.g. [link](https://blog.docker.com/2014/06/why-you-dont-need-to-run-sshd-in-docker/). my vote would be to leave it out.

Reply:
There's no reason to include ssh in a generic builder.